api/serverpb/server.proto

```diff
@@ -197,6 +197,8 @@ message Settings {
   bool azurediscover_enabled = 17;
   // True if the PMM instance is connected to Platform
   bool connected_to_platform = 19;
+  // Includes list of collected telemetry
+  repeated string telemetry_summaries = 20;
```

Comment:
Maybe it makes sense to create endpoint for that data, not sure. 🤔

Reply:
I guess better to use it here, because these summaries are used only on the Advanced settings page, and it's simpler to handle them on UI together with other data on this page.

I don't see any sense to move it to another endpoint.